src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Microsoft.DotNet.SourceBuild.SmokeTests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should we be using Arcade with this project?  Should it's output be going to the tarball artifacts directory?

Reply:
I think I'm on the side of leaving Arcade out of it to keep it simpler.  I think it would be convenient to have everything in the artifacts directory if it's straightforward but I don't think it's a blocker.

Reply:
I had attempted to do this earlier but ran into the following issue:

```
/repos/tarball-6.0.101/.dotnet/sdk/6.0.100/Microsoft.Common.CurrentVersion.targets(858,5): warning MSB3539: The value of the property "BaseIntermediateOutputPath" was modified after it was used by MSBuild which can lead to unexpected build results. Tools such as NuGet will write outputs to the path specified by the "MSBuildProjectExtensionsPath" instead. To set this property, you must do so before Microsoft.Common.props is imported, for example by using Directory.Build.props.
```

It get's a little messy which is why I asked about Arcade.  I propose we leave it as is for now if we don't like it or it proves to be an issue, we can refactor it later.

Reply:
That sounds like a good plan to me.